src/clusterfuzz/_internal/crash_analysis/crash_result.py

```diff
@@ -18,6 +18,7 @@
 from clusterfuzz._internal.base import utils
 from clusterfuzz._internal.crash_analysis import crash_analyzer
 from clusterfuzz._internal.crash_analysis.stack_parsing import stack_analyzer
+from src.clusterfuzz.stacktraces import CrashInfo
```

Comment:
This was causing the test failure. There shouldn't be a 'src' here.

Reply:
oh didn't catch that. Auto import on my vscode gives me headaches sometimes.